web/client/components/geostory/layouts/sections/Title.jsx

```diff
@@ -74,6 +97,13 @@ export default backgroundProp(({ id, background = {}, contents = [], add = () =>
                     // TODO: add
                     add('sections', id, SectionTypes.IMMERSIVE);
                 }
-            }]}/>}
+            }/*,
```

Comment:
I think you are going to override this change in the next PR. If you don't please remove this commented code. Otherwise leave

Reply:
yes it has changed in the meanwhile :)